tools/automator/automator.sh

```diff
@@ -280,6 +280,16 @@ merge() {
   fi
 }
 
+validate_changes_exist_in_prior_pr() {
```

Comment:
nit: Should we make a similar change here to address the commit instead of PR?

Reply:
Would it make more sense to call this `validate_changes_exist_in_latest_commit` or something like that? The use of prior here made me think we're looking back at history, which we aren't really.

Reply:
Updated the function name and added a description.

Reply:
Thanks :)